src/common/playerStore.ts

```diff
@@ -25,13 +25,11 @@ export const isVideoEnabledAtom = atom<boolean>(true);
 
 export const isAudioEnabledAtom = atom<boolean>(true);
 
-export const userMediaStreamAtom = atom<MediaStream>(new MediaStream());
+export const userMediaStreamAtom = atom<MediaStream | null>(global?.window ?  new MediaStream() : null);
 
 export const userMediaStreamErrorAtom = atom<string | null>(null);
 
-export const userNameAtom = atom<string>(
-  localStorage.getItem('playerName') || generatePlayerName(),
-);
+export const userNameAtom = atom<string>( generatePlayerName());
```

Comment:
```suggestion
export const userNameAtom = atom<string>(generatePlayerName());
```

I saw that you added the corresponding useEffect hook on the `UserPlayerSelfModel`. I assume you've tested it, but this change doesn't affect any client side behavior right? 

I don't see a place where we call `localStorage.setItem` other than when the `handleChange` function is called when the user types in the name field. So just wanted to double check that the default names functionality doesn't break.

Reply:
I'll be sure to QA this hotspot before merging  -- so far no indications I'm breaking this but will verify further